config/application.properties

```diff
@@ -101,6 +101,12 @@ spring.jpa.properties.hibername.use_sql_comments = true
 spring.jpa.properties.hibernate.format_sql = true
 spring.jpa.properties.hibernate.type = trace
 
+spring.jpa.properties.hibernate.cache.use_second_level_cache = true
+spring.jpa.properties.hibernate.cache.use_query_cache = true
+spring.jpa.properties.hibernate.cache.region.factory_class = org.hibernate.cache.ehcache.EhCacheRegionFactory
+
+spring.jpa.properties.javax.persistence.sharedCache.mode = ENABLE_SELECTIVE
```

Comment:
ENABLE_SELECTIVE instead of ALL allows us to specify not to cache on our classes if we want.